interface/Types.h

```diff
@@ -4,9 +4,9 @@
 #include <Math/Vector4D.h>
 #include <cp3_llbb/HHAnalysis/interface/Indices.h>
 
-typedef ROOT::Math::LorentzVector<ROOT::Math::PtEtaPhiE4D<float>> LorentzVector;
-
 namespace HH {
+    typedef ROOT::Math::LorentzVector<ROOT::Math::PtEtaPhiE4D<float>> LorentzVector;
+
```

Comment:
It's not directly related, but it'll help if / when using MoMEMta, which defines also a `LorentzVector`. I can move it back if you want